x-pack/plugins/siem/server/lib/uncommon_processes/types.ts

```diff
@@ -18,7 +18,11 @@ export interface UncommonProcessesAdapter {
 type StringOrNumber = string | number;
 export interface UncommonProcessHit extends Hit {
   total: TotalHit;
-  host: Array<{ id: string; name: string }>;
+  // I am fudging this a bit since it can be technically:
+  // host: Array<{ id: string | string[] | null | undefined; name: string | string[] | null | undefined }>;
+  // However, since GraphQL ToStringArray converts it I am leaving it as is.
+  // TODO: Re-look at this and see if I should add the extra code or not
+  host: Array<{ id: string[] | null | undefined; name: string[] | null | undefined }>;
```

Comment:
`UncommonProcessHit` is only used in `es_adapter` in parsing the response from Elasticsearch, so before it's passed off to GQL, correct? In that case, you'd want that explicit type including `string` since it's specific to the ES response, right?

Reply:
It is pulling that from Elastic Search so it can be an array or a string technically. Let me see if I can fix that one up without a lot of refactor